source/core/ut_suite_manager.pkb

```diff
@@ -677,7 +739,11 @@ create or replace package body ut_suite_manager is
     resolve_schema_names(l_paths);
 
     l_schema_paths := group_paths_by_schema(l_paths);
-
+    
+    -- TODO : use a_tags to filter out path. Should we do it before ? we dont know suites, if we do it after
```

Comment:
Needs to be discussed?

Reply:
That was a note before we have decided the suitepath will be a primary filter and tags will filter remaining code.